cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -3900,6 +3930,30 @@
                   "  vendor_cookbook\n",
                   "fi\n",
                   "cd /tmp\n",
+                  "\n",
+                  "masterServerStatus=$(aws cloudformation describe-stack-resource --stack-name ",
```

Comment:
What about using `Fn::Sub` to replace StackName and Region in the whole user-data script?

See **UserData Commands** section of the following documentation:
https://docs.aws.amazon.com/en_us/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-sub.html

(Then I know that is not part of this patch but we could also think to use `Fn::Join: ["\n", [...]]` to merge the multiple lines of the user-data script)